iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/NetworkXml.java

```diff
@@ -135,7 +135,7 @@ private static void writeExtensionNamespaces(Network n, ExportOptions options, X
                         .stream()
                         .filter(e -> canTheExtensionBeWritten(getExtensionXmlSerializer(options, e), networkVersion, options))
                         .map(extension -> (ExtensionXmlSerializer<?, ?>) getExtensionXmlSerializer(options, extension)))
-                .collect(Collectors.toSet());
+                .collect(Collectors.toCollection(LinkedHashSet::new));
```

Comment:
Is this modification linked to the PR? If it is for tests (order), you should condition this to the parameter "sorted" in XML export.

Reply:
Actually it's something @flo-dup did. I think it must be needed.

Reply:
This modification came from PR #2231, it appeared there only because the merge from main hadn't been done.